CHANGELOG.asciidoc

```diff
@@ -78,6 +78,7 @@ https://github.com/elastic/beats/compare/v6.5.0...6.x[Check the HEAD diff]
 *Metricbeat*
 
 - Collect custom cluster `display_name` in `elasticsearch/cluster_stats` metricset. {pull}8445[8445]
+- Test etcd module with etcd 3.3. {pull}9068[9068]
```

Comment:
I think we don't add entries for tests, as these doesn't make it into the product itself

Reply:
In this case I wanted to mention it as indication of coverage for this version, as changed in the docs too.

Reply:
But I'm ok with changing or removing it in a future PR as this is already in master, wdyt?